### PR TITLE
Fix tests for builtins

### PR DESCRIPTION
**Pull request by siteshwar**
* Fix test case for deleting a builtin
* Fix a crash while parsing arguments in source builtin
* Rename 'time' feature test under libast to 'systime'

### CODE REVIEW

**Comment by krader1961** [comment]
Just a FYI, @siteshwar, that I'm not going to look at any of your PRs after PR #136 until we have resolved why that one is causing a SIGSEGV on Travis but not our local systems.

**Comment by siteshwar** [comment]
All tests for builtins are fixed now.

File: src/cmd/ksh93/bltins/misc.c [hunk 1]
```diff
@@ -218,6 +218,11 @@ int    b_dot_cmd(int n,char *argv[],Shbltin_t *context)
 	struct checkpt buff;
 	Sfio_t *iop=0;
 	short level;
+	Optdisc_t disc;
```
Comment (krader1961) [inline src/cmd/ksh93/bltins/misc.c]: It looks like the `optinit()` macro is supposed to be used to do this initialization. But this is another example where we should replace the AST code with standard functions. In this case `getopt_long()`. So I'm going to let it stand.

**Comment by krader1961** [comment]
Merged.

**Pull request closed without merging**